content/blog/2019/10/2019-10-04-vscode-caseStudy.adoc

```diff
@@ -0,0 +1,71 @@
+---
+layout: post
+title: "JCasC JSONSchema Case-Study Using VSCode"
+opengraph:
+  image: /images/logos/JCasC/JCasC.png
+tags:
+- community-bridge
+- JCasC
+- VSCode
+- community
+author: sladyn98
+---
+
+=== JSON Schema and VSCode
```

Comment:
Any background needed on the configuration-as-code plugin here?

Reply:
+1. Some intro is needed

Reply:
What is the purpose of of this header now?

Reply:
Got rid of it :)